test/integration/visitor_can_view_items_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class VisitorCanViewItemsTest < ActionDispatch::IntegrationTest
+  test "visitor can view items test" do
+    item_1 = create_chip("Slotachips", 20, "Super yummy")
+    item_2 = create_chip("Trader Joe's BBQ", 15, "I'd trade slota for these!")
+    item_3 = create_chip("Dang Coconut", 17, "Dang, these are good")
+
+    visit '/chips'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.